src/landOnFunctions.js

```diff
@@ -1,57 +1,67 @@
-export {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax};
+export {
+    standardProperty,
+    railroad,
+    utility,
+    eventCard,
+    noEvent,
+    go,
+    goToJail,
+    incomeTax,
+    luxuryTax,
+};
 
 function standardProperty(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
+    }
+    else{
+        emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[this.houses]);
+    }
 
-  return this;
+    return this;
 }
 
 function railroad(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
+    }
+    else{
+        emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[this.houses]);
+    }
 }
 
 function utility(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this.position);
-  }
+    if(this.ownerID == null){
+        emitter.emit(`promptBuy${activePlayer.socketID}`, this.position);
```

Comment:
Ah, I've found why the clients weren't reading utility names.  I had changed how I was going to do it and missed this one.